config/locales/en.yml

```diff
@@ -2351,4 +2351,22 @@ en:
       title: Access not allowed
       body_html: Oops! Looks like you're trying to access a page or client you do not have access to. Please contact Partner Support at <a href="mailto:partnersupport@getyourrefund.org">partnersupport@getyourrefund.org</a> if you need to merge a duplicate client you do not have access to.
     home:
-      already_started: "Already started? Sign in"
\ No newline at end of file
+      already_started: "Already started? Sign in"
```

Comment:
The text for these messages is very similar to the successful submission message text from `en.messages.successful_submission.email_body`